tests/golden/check-ignored/check-ignored.bats

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bats
+
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+
+@test "Ignore file with broken xrefcheck annotation: full path" {
+  run xrefcheck --ignored ./to-ignore/broken_annotation.md
+
+  assert_output --partial "All repository links are valid."
+}
```

Comment:
For completeness, can you add a test for "directory path" with just `to-ignore` please?